lib/ui/window/key_data_packet.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_WINDOW_KEY_DATA_MESSAGE_H_
+#define FLUTTER_LIB_UI_WINDOW_KEY_DATA_MESSAGE_H_
+
+#include <string.h>
+#include <vector>
+#include <functional>
+
+#include "flutter/fml/macros.h"
+#include "flutter/lib/ui/window/key_data.h"
+
+namespace flutter {
+
+typedef std::function<void (bool)> KeyDataPacketCallback;
```

Comment:
This needs a comment. (I'm not even clear why it's here, since it's not used anywhere in this class.)